phases/phase-2/challenges/add-reaction.md

```diff
@@ -0,0 +1,105 @@
+# Add a reaction to post
+:::tip Note  
+This challenge allows you to receive **30 Desmos Tokens** upon completion.  
+
+Please note that in order to prevent spam, you will receive the reward just **for the first post** you react to. No tokens will be awarded for later posts.  
+:::  
+
+During the [Phase 1 Like challenge](../../phase-1/challenges/like-post.md) we saw how we could add a like to post to show everyone that we appreciated such post. 
```

Comment:
During the [Phase 1 Like challenge](../../phase-1/challenges/like-post.md) we saw how we could add a like to post to show everyone that we appreciated such post. 

->

We have replaced the "Like" action to a more enriched "Reaction" set. Below instruction will guide you to make your first reaction and earn the reward for this challenge.

Reply:
I think we can change it to 

> With the latest Desmos software upgrade, we have replaced the "Like" action with the "Reaction" one. This allows you to express a larger set of emotions regarding a post.  

As it might sound better. We also do not need to specify the below paragraph will guide the user as the name already tells so.